modules/ublog/src/main/UblogApi.scala

```diff
@@ -78,10 +78,29 @@ final class UblogApi(
 
   def latestPosts(blogId: UblogBlog.Id, nb: Int): Fu[List[UblogPost.PreviewPost]] =
     colls.post
-      .find($doc("blog" -> blogId, "live" -> true), previewPostProjection.some)
-      .sort($doc("lived.at" -> -1))
-      .cursor[UblogPost.PreviewPost](ReadPref.sec)
-      .list(nb)
+      .aggregateList(maxDocs = nb, _.sec): framework =>
+        import framework.*
+        Match($doc("blog" -> blogId, "live" -> true)) -> List(
+          AddFields(
+            $doc(
+              "isSticky" -> $doc(
+                "$cond" -> $doc(
+                  "if"   -> $doc("$ne" -> $arr($doc("$type" -> "$sticky"), "missing")),
```

Comment:
could use $ifNull here

Reply:
Strange, I did not find the right syntax in the first place (and just found this circumvented way).
I fixed it.